<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Homebells-MacBook-Pro-2.local" name="Test Class 2(failed)" tests="1" failures="1" timestamp="27 Aug 2019 10:57:39 GMT" time="15.404" errors="0">
  <testcase name="check_auto_offer_in_database_table" time="15.402" classname="caroobi.automatedOffer">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
	at junit.framework.Assert.fail(Assert.java:55)
	at junit.framework.Assert.assertTrue(Assert.java:22)
	at junit.framework.Assert.assertFalse(Assert.java:39)
	at junit.framework.Assert.assertFalse(Assert.java:47)
	at caroobi.automatedOffer.check_auto_offer_in_database_table(automatedOffer.java:291)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- check_auto_offer_in_database_table -->
</testsuite> <!-- Test Class 2(failed) -->
